talend-component-maven-plugin/pom.xml

```diff
@@ -181,22 +185,27 @@
               <goal>install</goal>
               <goal>run</goal>
             </goals>
-            <configuration>
-              <showErrors>true</showErrors>
-              <projectsDirectory>src/it</projectsDirectory>
-              <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
-              <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
-              <settingsFile>src/it/settings.xml</settingsFile>
-              <pomIncludes>
-                <pomInclude>*/pom.xml</pomInclude>
-              </pomIncludes>
-              <goals>
-                <goal>clean</goal>
-                <goal>verify</goal>
-              </goals>
-            </configuration>
           </execution>
         </executions>
+        <configuration>
```

Comment:
The configuration block was not at the right location, it was moot. 
Learned it the hard way banging my head on the wall trying to understand why the script variables didn't show up in the post build script.

This probably means that the whole configuration was useless before (only default conf).

Reply:
Good catch!